libs/feature/record/src/lib/map-view/map-view.component.ts

```diff
@@ -227,13 +243,22 @@ export class MapViewComponent implements AfterViewInit {
     this.selection = null
   }
 
-  getLayerFromLink(link: DatasetOnlineResource): Observable<MapContextLayer> {
+  getLayerFromLink(link: DatasetOnlineResource, style?: { href: string; name: string }): Observable<MapContextLayer> {
     if (link.type === 'service' && link.accessServiceProtocol === 'wms') {
       return of({
         url: link.url.toString(),
         type: 'wms',
         name: link.name,
       })
+    } else if (
+      link.type === 'service' &&
+      link.accessServiceProtocol === 'tms'
+    ) {
+      return of({
+        type: 'maplibre-style',
+        name: link.name,
+        styleUrl: style?.href || null,
```

Comment:
Does that work? I had the impression that we needed to implement a new `vector-tiles` layer in the geospatial sdk to show a TMS layer without a style

Reply:
I've not touched on this here yet.

Reply:
Now done in https://github.com/geonetwork/geonetwork-ui/pull/1213/commits/236f9023b3a674916b88db8b3fe6e09c5686e537